plasma_framework/test/src/exits/payment/PaymentInFlightExitable.test.js

```diff
@@ -94,6 +93,11 @@ contract('PaymentInFlightExitable', ([_, alice, bob, carol]) => {
             return new PaymentTransaction(IFE_TX_TYPE, [input], [output]);
         }
 
+        function createDepositTransaction(owner, amount, token = ETH) {
+            const output = new PaymentTransactionOutput(amount, addressToOutputGuard(owner), token);
+            return new PlasmaDepositTransaction(output);
```

Comment:
this is self is not enough to make it a `deposit tx`, in fact, what we really need is to make sure it is mined in a deposit block number (that's how contract checks it). Basically I think we need two blocks in test, one child chain block an one deposit block to make sure we hit both branch.